### DISCUSSION TITLE
Custom sprites of M.I.L.F.

### DISCUSSION THREAD

**Discussion by ghost** [KadeArchive/Kade-Engine#272] [Q&A] [answered]
I'm making a mod for me and my friends that adds friday night vibin and a few other mods. In the Vibin mod, M.I.L.F. has different sprites, and I got everything else to change but gf and the Background Dancers stay as their old sprites. How to I fix this?

**Comment by ghost** [comment]
Update: I fixed the dancers but gf is still broken

**Comment by DizzyJune** [comment, marked as answer]
Look for the bfCar.png and gfCar.png files. They use different sprite sheets for the car levels due to them being in different positions then normal.

**Reply by ghost** [reply]
What about the dancers? Nothing I've tried works

**Reply by DizzyJune** [reply]
Look for the limo folder and select this file (Your KadeEngine probably isn't in your local disk like mine is though)
![image](https://user-images.githubusercontent.com/74737360/116864694-43932580-abd6-11eb-861d-1c4963063b21.png)
